flowmachine/flowmachine/features/subscriber/unique_values_from_queries.py

```diff
@@ -24,6 +25,11 @@ class UniqueValuesFromQueries(Query):
     """
 
     def __init__(self, query_list: List[Query], column_names: List[str]):
+
+        for query in query_list:
+            if False in [name in query.column_names for name in column_names]:
```

Comment:
Perhaps more readable to do `if any(name not in query.column_names for name in column_names)`,
or `if not all(name in query.column_names for name in column_names)`? (I'm just being picky)

Reply:
Nah, makes sense. I was casting around for that but blanked on it >_>